### DISCUSSION TITLE
Family history questionnaire

### DISCUSSION THREAD

**Discussion by sanaashrafali** [now-i-know-my-abcd/docs#16] [Where in the World is…? (Help)]
For any of you who have covaried for effects of family history when looking at psychopathology in ABCD - is it better to use the family history questionnaire or the Adult Self Report?

**Comment by sanaashrafali** [comment]
I would guess it depends on whether you are just controlling for a history of yes/no diagnosis (then family history is probably easiest) or interested in/have a hypothesis about dimensional caregiver psychopathology or severity of symptoms (then ASR would have richer info). And I'm guessing ASR is just the participating caregiver and family history is more extensive? In which case family history is more relevant for a heritability-type question and ASR more relevant for a parenting-specific question.

**Comment by stapert** [comment]
i would go with Family History, as the ASR is just from 1 person, the Participating Parent/Guardian.
